### DISCUSSION TITLE
CSV export Arabic text is not right with UTF-8

### DISCUSSION THREAD

**Discussion by MohamedEmad-Data** [dbeaver/dbeaver#13922] [Q&A] [answered]
![ss1](https://user-images.githubusercontent.com/82875093/133122052-c34450c7-3904-4147-ad3a-50911167f240.PNG)
![ss2](https://user-images.githubusercontent.com/82875093/133122107-b2cf931c-e8d4-4b3b-9c70-40ad3e1bba12.PNG)
Hello,

I'm facing a problem with Arabic text when exporting a CSV file from a query result with setting of encoding UTF-8, the text not appearing correct as the result from the DBeaver UI.
Please check the attached images.

Thank you, and appreciate your support.

**Comment by Matvey16** [comment, marked as answer]
@MohamedEmad-Data Hello, this is a MS Excel problem. It won't be able to show some unicode characters corectly without BOM, so you need to use this option in your export 
![dbeaver_uxxbf1kwCV](https://user-images.githubusercontent.com/82543000/133571155-7c330e3a-8ce6-404b-aa4e-288c44e86193.png)

Or you can open your original CSV in notepad for example and those characters would be shown correctly

**Reply by MohamedEmad-Data** [reply]
Hello @Matvey16,
Its solved now. Thank you so much! :)
